src/main/java/io/reactivex/Completable.java

```diff
@@ -559,7 +558,7 @@ public void onSubscribe(Subscription s) {
      * @throws NullPointerException if flowable is null
      */
     @SchedulerSupport(SchedulerKind.NONE)
-    public static <T> Completable fromNbpObservable(final Observable<T> observable) {
+    public static <T> Completable fromObservable(final Observable<T> observable) {
```

Comment:
Should these kinds of `fromX(X)` methods change to something like `fromX(ConsmuableX)`? I think its sufficient to adopt the more generic api but still call out to `Observable` in the name and in the javadoc comment.